solr/conf/solrconfig.xml

```diff
@@ -1,27 +1,26 @@
-<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Sorry, but just for my own understanding, why was the encoding removed? Is this explicitly defined elsewhere for Solr as the default?